go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -212,6 +215,9 @@ func ConvertPodFailureToError(status v1.PodStatus) (code, message string) {
 					c.LastTerminationState.Terminated.ExitCode,
 					c.LastTerminationState.Terminated.Reason,
 					c.LastTerminationState.Terminated.Message)
+			} else if c.State.Terminated != nil && strings.Contains(c.State.Terminated.Reason, OOMKilled) {
+				code = OOMKilled
+				message += fmt.Sprintf("\r\nContainer [%v] terminated due to being OOMKilled", c.ContainerID)
```

Comment:
Should we add the c.State.Terminated.Reason to the message?

Reply:
I guess basically keep the message from the previous block, and just set the code as OOMKilled?

Reply:
sure